org.eclipse.winery.repository.rest/src/test/resources/entitytypes/artifacttypes/baobab_inital.json

```diff
@@ -0,0 +1,50 @@
+[
```

Comment:
The filename does not reflect its content. It is NOT the initial Baobab tree, but the list of all artifact types. Maybe use "list.json" or "list-at-COMMITID.json" - Replace COMMITID by the appropriate COMMITID.

Reply:
This comment applies to ALL OTHER baobab_initial.json files!